capsules/src/driver.rs

```diff
@@ -46,6 +46,7 @@ pub enum NUM {
     Humidity              = 0x60001,
     AmbientLight          = 0x60002,
     NINEDOF               = 0x60004,
+    Proximity             = 0x60003,
```

Comment:
This syscall value was previously assigned to the pressure sensor HIL, perhaps it would be better to use `0x60005`?
Also, make sure to update `doc/syscalls/README.md` with this value!

Reply:
Done and done.